### ISSUE TITLE
Files API upload does not seem to use file_name attribute

### ISSUE THREAD

**Issue by woutwoot** [HubSpot/hubspot-api-python#32]
When trying to upload a PNG file using the files API:

```
    file = hubspot.files.files.files_api.upload(file=open(image_path, 'rb').read(),
                                                file_name=os.path.basename(image_path),
                                                folder_path='folder',
                                                options=json.dumps({
                                                    "access": "PUBLIC_INDEXABLE",
                                                    "overwrite": True,
                                                    "duplicateValidationStrategy": "RETURN_EXISTING",
                                                    "duplicateValidationScope": "EXACT_FOLDER"
                                                }))
```

I get a bad request response:

```
"message":"Could not determine name from file"
```
While I clearly did specify a file name. I'm using version 3.6.1

**Comment by earnestjfk** [comment]
As a workaround, try changing [these](https://github.com/HubSpot/hubspot-api-python/blob/master/hubspot/files/files/api/files_api.py#L1517-L1518) lines from:

```
 if "file" in local_var_params:
    form_params.append(("file", local_var_params["file"]))  # noqa: E501
```

to:

```
if "file" in local_var_params:
    form_params.append(("file", ('integration-tests-would-have-caught-this', local_var_params["file"])))
```

**Comment by woutwoot** [comment]
@earnestjfk Thanks a lot! This does indeed solve the issue (nice hint there, haha). Hopefully HubSpot will pick this up.

**Comment by woutwoot** [comment]
@atanasiuk-hubspot Are you guys planning on fixing this issue?

**Comment by plaurynovich-hubspot** [comment]
Hi, @woutwoot. Thank you, sure, we're going to fix it.

**Comment by plaurynovich-hubspot** [comment]
Should be fixed in 3.7.2.

**Comment by woutwoot** [comment]
Thanks, this is fixed indeed!
